src/node/dishonest_node.cpp

```diff
@@ -143,7 +143,7 @@ void DishonestNode::handleVote(const Message& message, unsigned k) {
     }
     
     votes[k][epoch].insert(message.from());
-    if (3 * votes[k][epoch].size() >= 2 * numOfNodes) {
+    if (3 * votes[k][epoch].size() > 2 * numOfNodes) {
```

Comment:
This is good moment to add testcase when `nrOfNodes % 3 == 0`

Reply:
We can create a separate issue for it.

Reply:
oh sorry, didn't see that and added already

Reply:
no need for separate issue, this test belongs here